tools/rna_tools/pipmir/tool-data/twobit_update.loc.sample

```diff
@@ -0,0 +1,23 @@
+#This is a sample file distributed with Galaxy that is used by some
```

Comment:
Any reason you call it twobit_update and not twobit ... it should be exactly the same as the other tools that are using twobit files. Only then you can all use the same data_manager.